metadata.json

```diff
@@ -58,6 +58,6 @@
     }
   ],
   "pdk-version": "2.1.0",
-  "template-url": "https://github.com/puppetlabs/pdk-templates#2.1.0",
-  "template-ref": "tags/2.1.0-0-ga675ea5"
+  "template-url": "https://github.com/puppetlabs/pdk-templates#master",
+  "template-ref": "remotes/origin/master-0-ga58fd92"
```

Comment:
Are we using features of the head of the pdk-templates repo? `master` was renamed to `main` in that repo so this may be out of date.

Reply:
Ditto, iv used 2.1.0 in most releases, and MAIN is close enough, but master is significantly out of date

Reply:
Changed the ref to `main`.  Changes look fine and tests pass, so I'll include that change in a rebase and force push.